keystore/keystore_test.go

```diff
@@ -73,12 +84,54 @@ func TestEncryptAndDecryptFromFile(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = EncryptAndWriteToFile(filename, priv, password)
+	err = EncryptAndWriteToFile(file, priv, password)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	res, err := ReadFromFileAndDecrypt(fp, password)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer os.Remove(filename)
+
+	if !reflect.DeepEqual(priv, res) {
+		t.Fatalf("Fail: got %v expected %v", res, priv)
+	}
+}
+
+func TestEncryptAndDecryptFromFile_Sr25519(t *testing.T) {
+	filename := "./test_key"
+	password := []byte("noot")
+
+	fp, err := filepath.Abs(filename)
```

Comment:
You could save a lot of code by using helper functions for things like creating files

Reply:
I have added a helper func and cleaned up tests